### DISCUSSION TITLE
Can't extend brightness past the point currently set on display themself

### DISCUSSION THREAD

**Discussion by runxel** [MonitorControl/MonitorControl#1093] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

I'm a convert from Windows, where I have used TwinkleTray for regulating the brightness of the external display.
MonitorControl seems not to be able to adjust the brightness to levels beyond the current setting of the display, i.e. if the setting (physically set with buttons in a menu) of the display is currently 20%, than the MonitorControl's 100% will be that.

Also after moving the slider past 50% nothing happened anymore, but this was relieved by using the "combined dim switchpoint" (or whatever its called in english, in german its "Kombinierter Dimm-Umschaltpunkt") and turning it completely to the right. After this operation the gradient was completely usable.

### Expected behavior

It should be possible to have the brightness gradient match to the display.

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.4 Monterey
- Mac model: MacBook Pro (16", 2021)
- MonitorControl version: 4.1.0 Build 7034
- Monitor(s): Samsung U28E590 4K display
- Monitor Cable(s)/Connection(s): MacBook Pro -> Satechi USB-C Hub (ST-TXMM8PAM) -> Mini DisplayPort -> Samsung U28E590
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
Hi, hardware brightness control do not seem to work. Probably the hub is at fault (?) and does not relay DDC commands. Settings between 0-50% are software dimming by default and 50-100% is hardware (this is to be able to reach full dimming beyond the zero backlight level of the display).

**Reply by runxel** [reply]
Makes sense. So to test this I would need to hook the display directly to the MacBook, right?
Thing is, for whatever reason: Apple ditched their very own connector (Mini DP was their invention) and went with HDMI !?  Which is weird, I'd say, since DP is more potent than HDMI...
So, I'd need a USB-C to Displayport cable, if something like that even exists.... (?)

**Reply by waydabber** [reply]
Yes, you need an USB-C to DisplayPort cable or a dongle.  The HDMI port of the MacBook Pro also do not relay DDC commands so using a HDMI cable won't work.
